addon/mixins/icon.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+const { observer } = Ember;
+
+export default Ember.Mixin.create({
+
+  iconDidChange: observer('icon', function() {
```

Comment:
It may make more sense to add the observer in the DraggabilityMixin.

Thoughts?